packages/bridge/src/toast/toast.types.ts

```diff
@@ -0,0 +1,17 @@
+import { BridgeRequest } from "../messaging";
+import { BridgeMethod } from "../types";
+import { SwirlToastIntent } from "@getflip/swirl-components/loader";
+
+export type ShowToastRequestParams = {
+  content: string;
+  duration?: number;
```

Comment:
This doesn't fit with the mobile toast duration attribute, because there we have it as an enum `short`, `medium`, `long`.
We need to decide if we drop it for now, or if we make it compatible with both mobile and web.

Reply:
I think the ms approach is more flexible and explicit than the enum. (What does short mean?) Maybe Flip Mobile can map specific ranges to their enum values?

Reply:
We can use the ms approach. On the Dart side we can convert it into a Duration object and drop the enum approach.